go.mod

```diff
@@ -18,6 +20,7 @@ require (
 	golang.org/x/sync v0.9.0
 	golang.org/x/term v0.26.0
 	gopkg.in/yaml.v3 v3.0.1
+	pault.ag/go/debian v0.17.0
```

Comment:
There aren't really any `.deb` packages that are current; most are 7+ years old.